### PR TITLE
Fix properties regarding NightwatchLogEntry

### PR DESCRIPTION
**Pull request by benjaminr-ps**
...based on the following console log of an entry:

```
{
  level: { value: 800, name: 'INFO' },
  type: '',
  timestamp: 1661843309753,
  message: 'https://server/oidc/scripts/opi.js 67:10 "Unable to complete request from https://server"'
}
```

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Log API](https://nightwatchjs.org/api/getLog.html)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## nightwatch ([<kbd>unpkg</kbd>](https://unpkg.com/browse/nightwatch@latest/))
was missing the following properties:
1. createClient
1. client
1. cli
1. runTests
1. CliRunner

<details>
<summary>as well as these 2 other properties...</summary>
<p>initClient, runner</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 48d80bcd9c539147b68870e2cb5b339585f8aaa5
</p>

**Comment by garg3133** [comment]
@vaibhavsingh97 Can you please take a look at this, since you're the last person to work on this?

Considering that `NightwatchLogEntry` is only used for 2 commands in `index.d.ts` and both of these commands call [this method](https://github.com/nightwatchjs/nightwatch/blob/161c11e969f8393d68bc7c28a7e0bfb9f24455ba/lib/transport/selenium-webdriver/method-mappings.js#L163) which returns a specific object, we should have the interface `NightwatchLogEntry` in that format only. (Like `level` property of `NightwatchLogEntry` should only have `Level` interface and no literals or `number` type.

File: types/nightwatch/index.d.ts [hunk 1]
```diff
@@ -1698,7 +1698,7 @@ export interface Level {
     /**
```
Comment (vaibhavsingh97) [inline types/nightwatch/index.d.ts:1698]: In `Level` interface `name` should be change from `string` to `'ALL' | 'DEBUG' | 'FINE' | 'FINER' | 'FINEST' | 'INFO' | 'OFF' | 'SEVERE' | 'WARNING'`

File: types/nightwatch/index.d.ts [hunk 2]
```diff
@@ -1676,12 +1676,12 @@ export interface NightwatchLogEntry {
...
+    timestamp: number;
 
     /**
      * The log type, if known.
      */
-    opt_type?: string;
+    type?: string;
 
     /**
      * Severity level
```
Comment (vaibhavsingh97) [inline types/nightwatch/index.d.ts:1687]: In `NightwatchLogEntry`, `level` should be `level:  Level`

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by garg3133** [comment]
@benjaminr-ps Also, if you could add tests for these changes, that would be great :)

**Pull request closed without merging**